.github/workflows/on-pullrequest.yml

```diff
@@ -42,7 +42,8 @@ jobs:
       - name: Dependency Review ("Dependabot on PR")
         uses: actions/dependency-review-action@0ff3da6f81b812d4ec3cf37a04e2308c7a723730 # v3.0.2
 
-  analyze-code:
+  # CodeQL breaks if you rename this job
+  analyze:
```

Comment:
Trying to resolve this warning:

![image](https://user-images.githubusercontent.com/9324390/236037427-22464550-9c38-4343-99a8-0a6f8dba7928.png)